lib/isaac/isaac_types.hpp

```diff
@@ -185,18 +190,23 @@ BOOST_PP_REPEAT( 3, ISAAC_DIM_DEF, ~ )
 #undef ISAAC_DIM_TYPES
 #undef ISAAC_DIM_TYPES_DIM
 
+/**
+ * @brief Container for all simulation sizes
+ * 
+ * @tparam simdim 
+ */
 template < ISAAC_IDX_TYPE simdim >
 struct isaac_size_struct
 {
-    isaac_size_dim < simdim > global_size;
-    ISAAC_IDX_TYPE max_global_size;
-    isaac_size_dim < simdim > position;
-    isaac_size_dim < simdim > local_size;
-    isaac_size_dim < simdim > local_particle_size;
-    isaac_size_dim < simdim > global_size_scaled;
-    ISAAC_IDX_TYPE max_global_size_scaled;
-    isaac_size_dim < simdim > position_scaled;
-    isaac_size_dim < simdim > local_size_scaled;
+    isaac_size_dim < simdim > global_size;         //size of volume
+    ISAAC_IDX_TYPE max_global_size;                //? size if greatest dimension
```

Comment:
ping @theZiz @FelixTUD: what is this?

Reply:
Yes, this is the biggest of all dimensions of the volume above.

Reply:
resolved in a1e1270fd313a036d071ff56174d7fc40f1272a9